<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NotificationWidget</class>
 <widget class="QWidget" name="NotificationWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>315</width>
    <height>121</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Уведомление</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
   <item>
    <widget class="QLabel" name="labelThumbnail">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>100</height>
      </size>
     </property>
     <property name="text">
      <string>Thumbnail</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0">
       <item>
        <widget class="QLabel" name="labelTitle">
         <property name="text">
          <string>Title</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="labelIcon">
         <property name="text">
          <string>Icon</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
